flink-yarn/src/test/java/org/apache/flink/yarn/entrypoint/YarnEntrypointUtilsTest.java

```diff
@@ -132,37 +130,36 @@ public void testDynamicParameterOverloading() throws IOException {
         Configuration overloadedConfiguration =
                 loadConfiguration(initialConfiguration, dynamicParameters);
 
-        assertThat(
-                overloadedConfiguration.get(JobManagerOptions.JVM_METASPACE),
-                is(MemorySize.MAX_VALUE));
+        assertThat(overloadedConfiguration.get(JobManagerOptions.JVM_METASPACE))
+                .isEqualTo(MemorySize.MAX_VALUE);
     }
 
     @Nonnull
-    private static Configuration loadConfiguration(Configuration initialConfiguration)
-            throws IOException {
+    private Configuration loadConfiguration(Configuration initialConfiguration) throws IOException {
         return loadConfiguration(initialConfiguration, new HashMap<>());
     }
 
     @Nonnull
-    private static Configuration loadConfiguration(
+    private Configuration loadConfiguration(
             Configuration initialConfiguration, Configuration dynamicParameters)
             throws IOException {
         return loadConfiguration(initialConfiguration, dynamicParameters, new HashMap<>());
     }
 
     @Nonnull
-    private static Configuration loadConfiguration(
+    private Configuration loadConfiguration(
             Configuration initialConfiguration, Map<String, String> env) throws IOException {
         return loadConfiguration(initialConfiguration, new Configuration(), env);
     }
 
     @Nonnull
-    private static Configuration loadConfiguration(
```

Comment:
why are these no longer static? The tempBaseDir could be static as well.

Reply:
Updated

Reply:
the methods could still be static

Reply:
Done.